code/src/containers/products-container/ProductsContainer.test.tsx

```diff
@@ -100,65 +77,31 @@ describe("Test ProductsContainer", () => {
       products: []
     });
 
-    const possibleProduct = screen.queryByRole("link");
-    const skeletons = container.getElementsByClassName("spa-product-skeleton");
-
+    const possibleProduct = screen.queryByTestId("product-card");
     expect(possibleProduct).not.toBeInTheDocument();
+
+    const skeletons = container.getElementsByClassName("spa-product-skeleton");
     expect(skeletons.length).toBe(10);
   });
 
   test("Should accept and set className", () => {
     const { container } = renderProductsContainer({ className: "products" });
 
     const gridContainer = container.getElementsByClassName("products")[0];
-
     expect(gridContainer).toBeInTheDocument();
   });
 
   test("Should render error", () => {
     renderProductsContainer({ isError: true });
 
     const errorElement = screen.getByText("errors.somethingWentWrong");
-
     expect(errorElement).toBeInTheDocument();
   });
 
   test("Should render custom passed error", () => {
     renderProductsContainer({ isError: true, errorMessage: "error" });
 
     const errorElement = screen.getByText("error");
-
     expect(errorElement).toBeInTheDocument();
   });
```

Comment:
why this spaces was removed ? It was more readable

Reply:
Reverted it back, thanks